### ISSUE TITLE
Separate color for NonText

### ISSUE THREAD

**Issue by tiberiuichim** [AlessandroYorba/Despacio#11]
I've noticed that the NonText highlight is linked to Comment. 

I have something like this in my .vimrc:

    set listchars=tab:▸\ ,trail:•,extends:>,precedes:<,nbsp:+,eol:¬

Which makes the EOL show up and is highlighted with NonText. I don't want these characters to be as visible as my comments, I want them really dim, so I setup custom highlights for them.

I wonder if this wouldn't benefit more people, by having a separate color for the NonText highlight in the colorscheme. I have it set to ``#112233``

**Comment by AlessandroYorba** [comment]
@tiberiuichim Good point! I like your idea. Would you mind trying this in your vimrc. It uses a teal color alreay found in the palette. If you approve, I'll commit a change

```VimL
autocmd ColorScheme despacio highlight! NonText guifg=#5f8787 ctermfg=66 
```

**Comment by tiberiuichim** [comment]
It seems kind of light for me, but I usually set the background to something darker. It all depends on the amount of ambient lighting (and even machine, I use a different color on my desktop from my laptop), so there's probably no good answer here.

**Comment by AlessandroYorba** [comment]
@tiberiuichim I'm open to any color within the Despacio palette. How about the brown/tan color found in CursorLineNr:
```VimL
autocmd ColorScheme despacio highlight! NonText guifg=#af875f ctermfg=137
```

**Comment by tiberiuichim** [comment]
Still too light. The current comment color, I think, is the darkest, so there's no right answer from the Despacio palette.

**Comment by AlessandroYorba** [comment]
@tiberiuichim Yo! I revised despacio on the nightly build. I think it's a winner. If you can take a look and  tell me what you think https://github.com/AlessandroYorba/Despacio/tree/nightly

Also I remember you saying once that you use a transparent background. I'm thinking of adding a variable that removes background highlighting for terminal users. What other UI elements do you also like having no background for? ( e.g LineNr, TabLine, Normal )
